src/env.js

```diff
@@ -1,4 +1,3 @@
-import dotenv from 'dotenv';
+require('dotenv').config();
```

Comment:
There's a couple spots here were you swapped from CommonJS to esm or vice versa. 

I'm just curious why, and also I wanted to point out that [there's a ticket to possibly switch us completely over to CommonJS on node](https://ocio-jira.acf.hhs.gov/browse/TTAHUB-331)

Anyways, I think newer versions of Node will support esm out of the box, but it could be awhile before we get there. (#702)

Reply:
I had to convert all our models to CommonJS to get the importing in `src/models/index.js` to work. I agree it would be nice to land on CommonJS or ESM, but I am not knowledgeable enough on the differences to know which one to switch to (or how much work it will be)